src/main/kotlin/lotto/Lotto.kt

```diff
@@ -2,25 +2,24 @@ package lotto
 
 private const val NUMBERS_SIZE = 6
 
-class Lotto(val numbers: List<Int>) {
+class Lotto(numbersList: List<Int>) {
```

Comment:
If you let the primary constructor take a List<LottoNumber> and have the secondary constructor accept a List<Int>, you can avoid creating unnecessary internal variables.
It’ll keep the class cleaner and more focused! 😊